deep_speech_2/layer.py

```diff
@@ -0,0 +1,155 @@
+"""Contains DeepSpeech2 layers."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import paddle.v2 as paddle
+
+DISABLE_CUDNN_BATCH_NORM = True
```

Comment:
Why not make ```DISABLE_CUDNN_BATCH_NORM``` a parameter of ```conv_bn_layer```? If hard coding here, user has to modify this file when training on ```cpu``` mode.